keyboards/studiokestra/fairholme/keyboard.json

```diff
@@ -0,0 +1,242 @@
+{
+    "manufacturer": "Studio Kestra",
+    "keyboard_name": "Fairholme",
+    "maintainer": "Studio Kestra",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B11", "B2", "B1", "B0", "A7", "A6", "A5", "A4", "A3", "F1", "F0", "C15", "C14", "C13"],
+        "rows": ["A0", "A1", "A2", "A9", "A10"]
+    },
+    "processor": "STM32F072",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x6002",
+        "vid": "0x7C10"
+    },
+    "indicators": {
+        "caps_lock": "B10",
+        "on_state": 1
+    },
+    "community_layouts": ["60_ansi", "60_ansi_split_bs_rshift", "60_iso"],
```

Comment:
Could the following compatible community layouts be added?:
- 60_ansi_tsangan
- 60_ansi_tsangan_split_bs_rshift
- 60_ansi_wkl
- 60_ansi_wkl_split_bs_rshift
- 60_hhkb
- 60_iso_split_bs_rshift
- 60_iso_tsangan
- 60_iso_tsangan_split_bs_rshift
- 60_iso_wkl
- 60_iso_wkl_split_bs_rshift

_P.S. matching keymap(s) not required_